install/charts/dir/apiserver/values.yaml

```diff
@@ -64,6 +64,13 @@ config:
     # Datastore to use for routing persistance.
     # datastore_dir: ""
 
+    # GossipSub configuration for efficient label announcements
+    # When enabled, labels are propagated via GossipSub mesh to ALL subscribed peers
```

Comment:
When enabled we still use `DHT+Pull mechanism`, right? I think this could be clearer in this comment here